dev/breeze/src/airflow_breeze/commands/testing_commands_config.py

```diff
@@ -132,41 +123,6 @@
         ],
     },
     TEST_ENVIRONMENT_DB,
-    TEST_PARALLELISM_OPTIONS_XDIST,
```

Comment:
I found out some inconsistencies in test command parameters while I was adding new option - this one fixes it - the "task-sdk" command for now does not really use the parallel/db parameters - so that also removes them.